internal/command/instance_feature_model.go

```diff
@@ -0,0 +1,70 @@
+package command
+
+import (
+	"context"
+
+	"github.com/zitadel/zitadel/internal/domain"
+	"github.com/zitadel/zitadel/internal/errors"
+	"github.com/zitadel/zitadel/internal/eventstore"
+	"github.com/zitadel/zitadel/internal/repository/feature"
+)
+
+type FeatureWriteModel[T feature.SetEventType] struct {
+	eventstore.WriteModel
+
+	feature domain.Feature
+
+	Type T
```

Comment:
Shouldn't this attribute be "Value" or something? This "Type" seems rather confusing.

Reply:
renamed it in the write model and event type